prov/rxm/src/rxm.h

```diff
@@ -369,6 +375,20 @@ extern struct fi_domain_attr rxm_domain_attr;
 extern struct fi_tx_attr rxm_tx_attr;
 extern struct fi_rx_attr rxm_rx_attr;
 
+static inline void rxm_fastlock_empty(fastlock_t *lock)
+{
+}
+
+static inline void rxm_fastlock_acquire(fastlock_t *lock)
+{
+	fastlock_acquire(lock);
+}
+
+static inline void rxm_fastlock_release(fastlock_t *lock)
+{
+	fastlock_release(lock);
+}
```

Comment:
We're setting function pointers to these, so the static inline won't help.  We should set the pointers to  fastlock_*() directly, rather than jumping through another call.

Reply:
Yep, `static inline` is excessive here. But `fastlock_*()` is a macro in a non-debug case. It is impossible to use this as the function pointer.

Reply:
Moved these functions into rxm_ep.c file and drop `inline`